examples/resnet50_trt/Readme.md

```diff
@@ -0,0 +1,43 @@
+Description:
+This is an example to show how to use Python Backend to preprocess your image before using TensorRT to do inference. There will be an ensemble model include preprocess and resnet50_trt.
+
+You need to install numpy, pillow and torchvision by "pip install numpy pillow torchvision".
+
+Your model_repository should have a structure like this:
```

Comment:
Can you add the bash commands to create a directory structure that you have shown below? Similar to: https://github.com/triton-inference-server/python_backend/tree/main/examples/bls